lib/src/widgets/switch/ds_switch.widget.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSSwitch extends StatelessWidget {
+  /// Creates a customizable switch for use alone or in conjunction with ListTile
+  ///
+  /// Set the active and inactive colors using the parameters [activeColor]
+  /// and [inactiveColor], and use [isActive] to turn the switch on and off.
+  const DSSwitch({
+    Key? key,
+    required this.onChanged,
+    required this.isActive,
+    this.isDisabled = true,
+  }) : super(key: key);
+
+  /// Function callback to use when the [onChanged].
+  final ValueChanged<bool> onChanged;
+
+  /// Enables or disables the widget by changing the color of the switch.
+  final bool isDisabled;
+
+  /// Referring to [isActive] true ou false.
+  final bool isActive;
+
+  /// Referring to [inactiveColor] for  [isActive] false,
+  final Color inactiveColor = DSColors.neutralMediumSilver;
```

Comment:
Since this prop is only used inside this class, it should have the ```_``` prefix.

Reply:
At first it should not be private, because it is used in the switch, on the other hand, it would not need to be in the switchBase, I will fix it in the switch.